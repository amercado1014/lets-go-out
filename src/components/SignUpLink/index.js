import React from 'react';
import './styles.css';
import { Link, withRouter } from 'react-router-dom';

export const SignUpLink = () => (
  <p>
    {`Don't have an account?`}{" "}
    <Link className="signup-link" to='/signup'>
      Sign Up
    </Link>
  </p>
);

export default withRouter(SignUpLink);

