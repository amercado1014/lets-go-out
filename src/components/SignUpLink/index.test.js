import React from 'react';
import { SignUpLink } from '../SignUpLink';
import { shallow } from 'enzyme';

describe('SignUpLink', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SignUpLink />);
  });

  it('should math the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

