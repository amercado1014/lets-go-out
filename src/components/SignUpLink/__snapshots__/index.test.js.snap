// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpLink should math the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Don't have an account?",
        " ",
        <Link
          className="signup-link"
          replace={false}
          to="/signup"
>
          Sign Up
</Link>,
      ],
    },
    "ref": null,
    "rendered": Array [
      "Don't have an account?",
      " ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Sign Up",
          "className": "signup-link",
          "replace": false,
          "to": "/signup",
        },
        "ref": null,
        "rendered": "Sign Up",
        "type": [Function],
      },
    ],
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Don't have an account?",
          " ",
          <Link
            className="signup-link"
            replace={false}
            to="/signup"
>
            Sign Up
</Link>,
        ],
      },
      "ref": null,
      "rendered": Array [
        "Don't have an account?",
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Sign Up",
            "className": "signup-link",
            "replace": false,
            "to": "/signup",
          },
          "ref": null,
          "rendered": "Sign Up",
          "type": [Function],
        },
      ],
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
